packages/plugins/nodejs-react-json-tree/nodejs-react-json-tree.spec

```diff
@@ -0,0 +1,110 @@
+%global npm_name react-json-tree
+%global enable_tests 0
```

Comment:
This is not needed in bundling because we drop the whole `enable_tests`. See below.